### DISCUSSION TITLE
0.33.1 Beta Bug Report

### DISCUSSION THREAD

**Discussion by intitni** [intitni/CopilotForXcode#524] [Announcements]
<img alt="instruction" src="https://raw.githubusercontent.com/intitni/intitni/a8b3dae8a3a3118fc22913dde331f229a0b762af/copilot-for-xcode-update-instruction.svg"  height="30">

- Bump GitHub Copilot language server to 1.32.0. This version does not support the old versions of the language server, please make sure to update the language server in app and restart the service app. Some breaking changes are introduced, please let me know if it's still working correctly for you.
- Bump Codeium language server to 1.8.8.
- Support loading GitHub Copilot language server with certificates from the keychain.
- Support switching open chat mode. If you prefer to use the web chat gpt or something else, you can choose to open it in the browser or the in-app browser tab when open chat command is hit.
- Update TCA.
- Refactor GitHub Copilot and Codeium support for future updates.

If you have found any bug in this version, please report to the release discussion.

<hr /><em>This discussion was created from the release <a href='https://github.com/intitni/CopilotForXcode/releases/tag/0.33.1.beta'>0.33.1 beta</a>.</em>

**Comment by zenangst** [comment]
Just installed the latest beta and just want to let you know that `dimming typed text` looks and feels fantastic! Kudos! ❤️
